README.md

```diff
@@ -25,82 +56,70 @@ The API version is specified through the accept header, as follows:
 Accept:application/json;version=5
 ```
 
-The only currently acceptable version is 5
+The current accepted versions are:
+* 5
+* 6 - "one shot API"
```

Comment:
It is probably worth mentioning that 'v6' is controlled using the 'v6' endpoint rather than the Accept header